src/apis/impl/UserApiImpl.ts

```diff
@@ -88,6 +94,45 @@ export class UserApiImpl implements UserApi {
     return this.userApi.matchUsersBy(FilterMapper.toAbstractFilterDto<User>(filter, 'User'));
   }
 
+  filteredContactsFromAddresses(addresses: Array<Address>, telecomType: TelecomTelecomTypeEnum, addressType?: AddressAddressTypeEnum): Telecom | undefined {
```

Comment:
As this PR is related to MaintenanceTask mapping, this should not be present in this PR. 
Plus, it will be replaced by the createAndInviteUser service API, that will take care of : 
- Creating a user; 
- Sending him an email with its credentials; 
Patient will already exist and will be provided by argument